scipy-1.0/paper.tex

```diff
@@ -608,9 +608,11 @@ \subsection*{SciPy Matures}
 For example, a three-part minisymposium organized for CSE 2009 was
 featured in SIAM News\cite{siamcse09}.
 
-When Paul edited the CiSE May/June 2007 special issue, Python already had a
-strong enough presence in science and engineering so that editors solicited
-the special issue, but it was still niche enough that the average reader
+When Paul Dubois edited the May/June 2007 special issue of the 
```

Comment:
I agree that improvement is needed here, but the proposed change complicates the sentence structure. Perhaps:

In 2007, Python had a strong enough presence in science and engineering that the editors of IEEE Computing in Science and Engineering solicited a special issue, edited by Paul Dubois.  However, Python was still niche enough that the average reader...

Reply:
Revised accordingly. There are still three mentions of this same special issue in the span of a few paragraphs, which could probably be condensed to a single mention.